client/src/components/ProjectWizard/WizardSidebar/EarnedPointsMetContainer.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import { createUseStyles, useTheme } from "react-jss";
+
+const useStyles = createUseStyles({
+  pointsNotification: {
+    border: "2px solid transparent",
+    backgroundColor: "#a7c539",
+    color: "#0F2940",
+    boxShadow: "0px 0px 5px rgba(0, 0, 0, .2)",
+    marginTop: "16px",
+    maxWidth: "250px",
+    display: "flex",
+    justifyContent: "center",
+    cursor: "pointer",
```

Comment:
not sure what this pointer is supposed to signal to the user. pointers usually mean it's clickable but this container is not.

Reply:
Got rid of pointer